### ISSUE TITLE
1.7.0 crashes immediately because of missing dependencies

### ISSUE THREAD

**Issue by olejnjak** [tuist/tuist#1249]
**Describe the bug**
After running `tuist update` all tuist commands fail with following error.

```
olejnjak@Jakub-MacBook ~ % tuist update
Checking for updates...
Installing new version available 1.7.0
Downloading version 1.7.0
Installing...
Version 1.7.0 installed

olejnjak@Jakub-MacBook ~ % tuist
dyld: Library not loaded: /Users/natanrolnik/Documents/Open Source/tuist/.build/x86_64-apple-macosx/release/libSwiftToolsSupport.dylib
  Referenced from: /usr/local/bin/tuist
  Reason: image not found
zsh: abort      tuist
```

**To Reproduce**
1. update Tuist to version 1.7.0 - `tuist update`
2. run `tuist`

**Expected behavior**
No crash

**Desktop (please complete the following information):**

macOS Catalina 10.15.4

**Comment by fortmarek** [comment]
Thanks for reporting this! This has already been solved in [1.7.1](https://github.com/tuist/tuist/releases/tag/1.7.1). Unfortunately, this might necessitate some extra steps now that you have this "dirty" binary:
I'd first try: `bash <(curl -Ls https://install.tuist.io)`
If that doesn't help, try these steps:
`rm ~/.tuist/Versions/1.7.0`
`rm /usr/local/bin/tuist`
`bash <(curl -Ls https://install.tuist.io)`

Hopefully, this should work, let us know if it doesn't 🙂

**Comment by olejnjak** [comment]
Running [this uninstall script](https://github.com/tuist/tuist/blob/master/script/uninstall) worked, just `-r` was missing in the initial command
